sdlarch.c

```diff
@@ -579,6 +581,101 @@ static uintptr_t core_get_current_framebuffer() {
     return g_video.fbo_id;
 }
 
+/**
+ * Retrieve the current time in milliseconds.
+ *
+ * @todo Retrieve the unix timestamp in milliseconds?
+ * @return retro_time_t The current time in milliseconds.
+ */
+static retro_time_t core_get_time_usec() {
```

Comment:
`core_get_time_usec()` is not needed as you can use `cpu_features_get_time_usec()` which already returns time in usecs.

Reply:
Makes sense :+1: